src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/StringParsingHelpers.Statistics.cs

```diff
@@ -417,23 +417,23 @@ internal static IPInterfaceStatisticsTable ParseInterfaceStatisticsTableFromFile
 
                         return new IPInterfaceStatisticsTable()
                         {
-                            BytesReceived = ParseUInt64AndClampToUInt32(pieces[1]),
-                            PacketsReceived = ParseUInt64AndClampToUInt32(pieces[2]),
-                            ErrorsReceived = ParseUInt64AndClampToUInt32(pieces[3]),
-                            IncomingPacketsDropped = ParseUInt64AndClampToUInt32(pieces[4]),
-                            FifoBufferErrorsReceived = ParseUInt64AndClampToUInt32(pieces[5]),
-                            PacketFramingErrorsReceived = ParseUInt64AndClampToUInt32(pieces[6]),
-                            CompressedPacketsReceived = ParseUInt64AndClampToUInt32(pieces[7]),
-                            MulticastFramesReceived = ParseUInt64AndClampToUInt32(pieces[8]),
-
-                            BytesTransmitted = ParseUInt64AndClampToUInt32(pieces[9]),
-                            PacketsTransmitted = ParseUInt64AndClampToUInt32(pieces[10]),
-                            ErrorsTransmitted = ParseUInt64AndClampToUInt32(pieces[11]),
-                            OutgoingPacketsDropped = ParseUInt64AndClampToUInt32(pieces[12]),
-                            FifoBufferErrorsTransmitted = ParseUInt64AndClampToUInt32(pieces[13]),
-                            CollisionsDetected = ParseUInt64AndClampToUInt32(pieces[14]),
-                            CarrierLosses = ParseUInt64AndClampToUInt32(pieces[15]),
-                            CompressedPacketsTransmitted = ParseUInt64AndClampToUInt32(pieces[16]),
+                            BytesReceived = long.Parse(pieces[1]),
```

Comment:
In current master, it does `Math.Min(uint.MaxValue, ulong.Parse(pieces[1]))`; would it make sense to switch to `ulong`, instead of `long`?

Reply:
Yeah. Like this:
```cs
Math.Min(long.MaxValue, ulong.Parse(pieces[1]))
```

Reply:
Did this change